pages/linux/apt.md

```diff
@@ -4,28 +4,24 @@
 
 - Update list of packages and versions available. This should be run before running further apt commands:
 
-`apt update`
+`sudo apt update`
```

Comment:
Yep, @agorf is right. The contribution guidelines mention it - if a command requires `sudo`, it should be mentioned.